assets/js/filter_players_list.js

```diff
@@ -0,0 +1,18 @@
+const handleSportChange = e => {
+  const allTables = document.querySelectorAll(".fantasy-player-collection");
+  allTables.forEach(table => (table.style.display = "none"));
+
+  const sportAbbrev = e.target.value;
+  if (sportAbbrev) {
+    const sportTable = document.getElementById(sportAbbrev);
+    sportTable.style.display = "";
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)